vite.config.ts

```diff
@@ -1,53 +1,69 @@
+                    wrapperStatemachine: path.resolve(__dirname, '/packages/examples/wrapper_statemachine.html'),
+                    wrapperLangium: path.resolve(__dirname, '/packages/examples/wrapper_langium.html'),
+                    wrapperTs: path.resolve(__dirname, '/packages/examples/wrapper_ts.html'),
+                    wrapperWebSocket: path.resolve(__dirname, '/packages/examples/wrapper_ws.html'),
+                    wrapperAdvanced: path.resolve(__dirname, '/packages/examples/wrapper_adv.html'),
+                    reactPython: path.resolve(__dirname, '/packages/examples/react_python.html'),
+                    reactStatemachine: path.resolve(__dirname, '/packages/examples/react_statemachine.html'),
+                    reactTs: path.resolve(__dirname, '/packages/examples/react_ts.html'),
+                    workers: path.resolve(__dirname, '/packages/examples/workers.html'),
+                    verifyWrapper: path.resolve(__dirname, '/packages/examples/verify_wrapper.html'),
+                    verifyAlt: path.resolve(__dirname, '/packages/examples/verify_alt.html')
+                }
             }
-        }
-    },
-    resolve: {
-        alias: {
-            path: 'path-browserify'
-        }
-    },
-    server: {
-        origin: 'http://localhost:20001',
-        port: 20001
-    },
-    optimizeDeps: {
-        esbuildOptions: {
-            plugins: [
-                // copied from "https://github.com/CodinGame/monaco-vscode-api/blob/run-ext-host-in-worker/demo/vite.config.ts"
-                {
-                    name: 'import.meta.url',
-                    setup({ onLoad }) {
-                        // Help vite that bundles/move files without touching `import.meta.url` which breaks asset urls
-                        onLoad({ filter: /default-extensions\/.*\.js/, namespace: 'file' }, args => {
-                            let code = readFileSync(args.path, 'utf8');
-                            code = code.replace(
-                                /\bimport\.meta\.url\b/g,
-                                `new URL('/@fs/${args.path}', window.location.origin)`
-                            );
-                            return { contents: code };
-                        });
+        },
+        resolve: {
+            dedupe: ['monaco-editor', 'vscode']
+        },
+        server: {
+            origin: 'http://localhost:20001',
+            port: 20001
+        },
+        optimizeDeps: {
+            esbuildOptions: {
+                plugins: [
+                    // copied from "https://github.com/CodinGame/monaco-vscode-api/blob/main/demo/vite.config.ts"
+                    {
+                        name: 'import.meta.url',
+                        setup({ onLoad }) {
+                            // Help vite that bundles/move files in dev mode without touching `import.meta.url` which breaks asset urls
+                            onLoad({ filter: /.*\.js/, namespace: 'file' }, async args => {
```

Comment:
This is the import.meta workaround right?

Reply:
Exactly.